streamer/src/nonblocking/quic.rs

```diff
@@ -720,19 +692,17 @@ async fn handle_connection(
     stats.total_connections.fetch_add(1, Ordering::Relaxed);
     let mut max_streams_per_throttling_interval =
         stream_throttle::max_streams_for_connection_in_throttling_duration(
-            peer_type,
             params.stake,
             params.total_stake,
             stream_load_ema.clone(),
         );
-    let staked_stream = matches!(peer_type, ConnectionPeerType::Staked) && params.stake > 0;
```

Comment:
We can reduce some of this change by defining:

let staked_stream = params.stake > 0;

Reply:
Isn't it better to have this check than declaring a new stack variable? The change is only three lines. So review should be simple.